src/main/java/org/apache/rocketmq/dashboard/model/request/AclRequest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rocketmq.dashboard.model.request;
+
+import org.apache.rocketmq.common.PlainAccessConfig;
+
```

Comment:
Instead of adding getters/setters/toString manually, Lombok `@Data` can be used.
~public PlainAccessConfig getConfig() {~
~return config;~
~}~
```java
@Data
public class AclRequest {

```

Reply:
Oh oh, Nice,   because I wrote these code in the year before last, I forgot whether it had import lombok on that time, so these code maybe not  elegant or simple.  I will adjust later again.